api/lib/domain/usecases/import-schooling-registrations-from-siecle.js

```diff
@@ -1,31 +1,39 @@
 const { FileValidationError, SameNationalStudentIdInOrganizationError, SameNationalStudentIdInFileError } = require('../errors');
-const _ = require('lodash');
+const fs = require('fs').promises;
+const bluebird = require('bluebird');
+const { CHUNK_SIZE_SCHOOLING_REGISTRATION_DATA_PROCESSING } = require('../../infrastructure/constants');
```

Comment:
Il n'y a qu'un seul endroit où on utilise les chunks, je propose donc de raccourcir en 
```suggestion
const { SCHOOLING_REGISTRATION_CHUNK_SIZE} = require('../../infrastructure/constants');
```